application/go_inc/helpers/helper__functions.php

```diff
@@ -78,7 +78,17 @@ function go_get_menu_items($menu_id) {
         foreach($query->result() as $result){
             if($count == 1) {
                 $return .= "<div id='menu-id-" . $result->MenuID . "'>"; // setup parent DIV and UL
-                $return .= "<ul>"; 
+
+                //$return .= "<ul>"; 
+                
+                if($menu_id == 1){
```

Comment:
I like this approach, but we need to make it dynamic to accept more than two menus.  I suggest remove this if/else and replace with:

`$return .= "<ul class='menu-ul-" . $menu_id . ">";`

Also will need to use CSS3 for any css you might have added against this.  So a wildcart like https://stackoverflow.com/questions/5110249/wildcard-in-css-for-classes